Demo/Demo/Enums/DemoSettings.swift

```diff
@@ -0,0 +1,23 @@
+import UIKit
+
+enum DemoSettings {
```

Comment:
```suggestion
struct DemoSettings {
```

I don't think enum makes the most sense here

Reply:
Yeah so initially had this as a struct but was getting the following linter warning: `Types used for hosting only static members should be implemented as a caseless enum to avoid instantiation. (convenience_type)`. I did a bit more reading on caseless enums (I liked [this article](https://www.swiftbysundell.com/articles/powerful-ways-to-use-swift-enums/) that mentions it a bit down but there are some others) and I think it makes a lot of sense for the usecase of the settings as we use them today.